c2corg_ui/static/js/feed.js

```diff
@@ -140,27 +228,134 @@ app.FeedController.prototype.getDocumentsFromFeed = function() {
 
 
 /**
+ * get lastest topics
+ * @export
+ */
+app.FeedController.prototype.getDocumentsFromForum = function() {
+
+  this.busyForum = true;
+  this.api.readForum().then(function(response) {
+    this.handleForum(response);
+  }.bind(this), function() { // Error msg is shown in the api service
+    this.busyForum = false;
+    this.errorForum = true;
+  }.bind(this));
+};
+
+
+/**
  * Handles feed processing for Feed.js and Whatsnew.js
  * @param response
  * @public
  */
 app.FeedController.prototype.handleFeed = function(response) {
+
   this.error = false;
   this.busy = false;
   var data = response['data']['feed'];
   var token = response['data']['pagination_token'];
   this.nextToken = token;
-  for (var i = 0; i < data.length; i++) {
-    this.documents.push(data[i]);
+  if (window.innerWidth >= 1360) {
+    this.nbcolumn = 2;
+
+    for (var i = 0; i < data.length / 2; i++) {
+      this.documentsL.push(data[i]);
+      this.documents.push(data[i]);
+    }
+    for (var j = data.length / 2; j < data.length; j++) {
+      this.documentsR.push(data[j]);
+      this.documents.push(data[j]);
+    }
+
+  } else {
+
+    this.nbcolumn = 1;
+    for (var k = 0; k < data.length; k++) {
+      this.documentsL.push(data[k]);
+      this.documents.push(data[k]);
+    }
   }
-  // reached the end of the feed - disable scroll
-  if ((token && data.length === 0) || (!token && this.documents.length > 0)) {
+
+  if ((token && data.length === 0) || !token && this.documentsL.length > 0) {
     this.feedEnd = true;
   } else if (data.length === 0) { // first fetch with no feed returned.
     this.noFeed = true;
   }
+
+};
+
+/**
+ * Handles forum processing for Feed.js
+ * @param response
+ * @public
+ */
+app.FeedController.prototype.handleForum = function(response) {
+  this.errorForum = false;
+  this.busyForum = false;
+  var data = response['data'];
+  var postersAvatar = {};
+  if (data['users'] !== undefined) {
+    for (var j = 0; j < data['users'].length; j++) {
+      postersAvatar[data['users'][j]['username']] = data['users'][j]['avatar_template'].replace('{size}','24');
+    }
+
+    for (var i = 0; i < data['topic_list']['topics'].length; i++) {
+
+      data['topic_list']['topics'][i]['avatar_template'] = postersAvatar[data['topic_list']['topics'][i]['last_poster_username']];
+
+      this.forums.push(data['topic_list']['topics'][i]);
+      if (i == 15) {
+        break;
+      }
+    }
+  }
+};
+
+/**
+ * @param {Object} doc
+ * @return {string | undefined}
+ * @export
+ */
+app.FeedController.prototype.createURL = function(doc) {
+
```

Comment:
The current feed use the card directive defined at https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/card.js which indeed has a ``createURL`` function https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/card.js#L163